docs/extras/advanced-targeting/self-targeting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Self-Targeting
+sidebar_position: 2
+---
+
+Self-Targeting is a new DevCycle QA & debugging feature that allows you to easily serve yourself different variations of a feature without having to modify targeting rules in each environment. To leverage this feature, you must set up your DevCycle Identity by associating an SDK User ID to your DevCycle user profile. 
+
+![Self-Targeting Slideout on Feature Page](/aug-2023-self-targeting-slideout-open-feature-form.png)
+
+On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for feature, and how many other team-members within the current project have self-targeting overrides set as well. 
+
+Within the slide-out, you can setup your DevCycle Identity, select Self-Targeting overrides, and see the other Team Members that are currently self-targeting for that feature, including the variation and environment they have active overrides in. 
+
+## DevCycle Identity
+
+Setup your DevCycle Identity by entering the user ID you are providing to the DevCycle SDK, and which uniquely identifies you. Doing so unlocks debugging features such as Self-Targeting, a feature that allows you to override targeting rules to deliver different variations of the feature to yourself. 
```

Comment:
should there be an example of what this means exactly? Like maybe a code snippet showing the userId being passed to the SDK, and more of an explanation of what we mean by "your user ID"?

Reply:
@ajwootto that'd be great! who can help me with that?

Reply:
@ajwootto i'll add this later but get the docs out there today!